lib/script.js

```diff
@@ -49,6 +49,19 @@ export const config = {
       'Directory of the script',
     ],
   },
+  position: {
+    title: 'Panel position',
+    description: 'Position of the panel with script output. '
+        + '(Changes to this value will be applied upon reopening the panel.)',
+    type: 'string',
+    default: 'Bottom',
+    enum: [
+      'Top',
+      'Bottom',
+      'Left',
+      'Right',
```

Comment:
Could you use lowercase options so you don't have to use a `switch-case`?

Reply:
Yes, I can but then the options would be lowercase in the settings UI too. Is this desirable? For example, the options for `cwdBehavior` in this package are capitalized as well. My intention was consistency.

Reply:
I think that's fine. We don't need to be consistent here.

Reply:
OK, will do.

Reply:
I have one question though: Should I implement some kind of check that the value in the settings is one of the four allowed values? Or is the value obtained from `atom.config` guaranteed to be one of the values listed in the enum (or empty)?

Reply:
No, Atom itself creates a dropdown and only those will be allowed. So, there is no extra check needed.